.azure_pipelines/job_templates/olive-build-doc-template.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       displayName: Checkout release branch
       condition: ne('${{parameters.doc_version}}', 'latest')
 
-    - script: make install-olive INSTALL_EXTRAS=[cpu]
+    - script: pip install .[cpu]
```

Comment:
can you make all the `pip` commands into `python -m pip` to be safe?

Reply:
will do